### PR TITLE
Technical Steering Committee Charter,  GOVERNANCE,  etc.

### PR DESCRIPTION
**Pull request by jminor**
This change adds the OTIO Technical Steering Committee Charter.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (meshula) [inline CONTRIBUTING.md:76]:
```suggestion
Please make sure that your pull requests are clean. In other words, address only the issue at hand. Split minor corrections, formatting clean ups, and the like into other PRs. Ensure that new work has coverage in a test. Ensure that any resultant behavioral changes in other parts of the library are called out. Use the rebase and squash git facilities as needed to ensure that the pull request does not contain non-informative remnants of old or superseded work.
```

File: GOVERNANCE.md
Comment (meshula) [inline GOVERNANCE.md:9]:
```suggestion
There are three primary project roles: 
   - Contributors submit code to the project; 
   - Committers approve code to be included into the project; and
   - the Technical Steering Committee (TSC) provides overall high-level
project guidance.
```

File: GOVERNANCE.md
Comment (meshula) [inline GOVERNANCE.md:20]:
```suggestion
submits code, documentation, or other artifacts to the
project. Such contributions must be approved by a project
```

File: GOVERNANCE.md
Comment (meshula) [inline GOVERNANCE.md:68]:
```suggestion
If you are interested in becoming a Committer, contact the TSC at
https://lists.aswf.io/g/otio-discussion
```

File: GOVERNANCE.md
Comment (meshula) [inline GOVERNANCE.md:102]:
```suggestion
serves as the OpenTimelineIO representative on the Academy Software
Foundation (ASWF) Technical Advisory Council (TAC), and represents the 
project at ASWF TAC meetings.
```

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:13]:
```suggestion
    * **i.** Contributors include anyone in the technical community that contributes code, documentation, or other artifacts to the Project;
```
A piece of paper with a logo drawing on it is a non-technical artifact that could be contributed to the project. It may be scanned, producing a technical artifact but the piece of paper itself would be the originating contribution.

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:14]:
```suggestion
    * **ii.** Committers are Contributors who have been granted the ability to modify ("commit") source code, documentation or other technical artifacts in a project's repository; and
```

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:26]: "Series Manager" is not defined in this document, I am vague as to who that is, I think it should be stated. Their role however is seemingly explicit in the several mentions in this document.

**Review by meshula — CHANGES_REQUESTED** [review]

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (jminor) [reply tsc/OpenTimelineIO_TSC_Charter.md:13]: I don't think the word "technical" is trying to distinguish physical vs digital. Though, the project can only accept digital contributions in practice. I think the word "technical" is meant to match the meaning of "Technical Steering Committee" which is that the TSC makes technical decisions for a technology project.

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (jminor) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: I agree that "granted" is more clear, in that the TSC must actually grant that role to someone.

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [reply tsc/OpenTimelineIO_TSC_Charter.md:13]: Yes, that makes sense to me.

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [reply tsc/OpenTimelineIO_TSC_Charter.md:26]: Ah, I thought I recalled that this had been raised before. I feel the rest of the issues mentioned in #6, related to LF definitions and entities have been resolved in this current charter, with the exception of the Series Manager. In the context of this charter's current text, I don't know what a "Series" is, but I do know that a "Series Manager" helps coordinate resolving things. Since the series manager coordinates, as opposed to approving, blocking, or otherwise gating the activity of the TSC, the lack of clarity around the role does not seem to impede the business of the TSC. @jmertic, @jminor, does that sound right? If I've understood it properly, I'm happy to mark this conversation as "resolved, fine as is".

Comment (jmertic) [reply tsc/OpenTimelineIO_TSC_Charter.md:26]: Correct @meshula - the Series Manager is more of a legal role and does not impact day-to-day operations

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (jmertic) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: So it's true that it must be 'granted', but note that the TSC must have non-discriminatory criteria for becoming a committer. 'Earned' is a good word to use to indicate the burden of hitting the requirements is on the contributor and still gives the TSC the ability to vote to approve the new committer. Some projects have more liberal rules, so this wording gives the project more flexibility it's committer policies. Let me know if this helps.

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [reply tsc/OpenTimelineIO_TSC_Charter.md:26]: Thanks!

File: tsc/OpenTimelineIO_TSC_Charter.md
Comment (meshula) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: I see! My objection to "earned" is that it is not defined anywhere, and is therefore a subjective, not objective word. I snag on subjective words in contracts.... Effectively "earned" is a granting subject to possibly opaque TSC consideration, as opposed to any kind of measurable, non-discriminatory, accumulation of merit which "earn" implies.

Comment (jmertic) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: Yep - does that address your concern?

Comment (jminor) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: Re-reading this, I see that 2.c.iii. unambiguously explains that Committers are added/removed by majority approval of existing Committers. This addresses the "granted" part in my mind.
With that out of the way, the word "earned" in 2.c.ii. just serves to explain the difference between a Contributor and a Committer, not the mechanism for how to become one. Furthermore, 2.c says "Unless otherwise documented" and we have a whole section on GOVERNANCE.md titled "How to Become a Committer" which covers that.

Comment (jmertic) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: OK - seeing if there's anything open on this still. Sorry to bug ;-)

Comment (meshula) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: all good, thanks!

Comment (jmertic) [reply tsc/OpenTimelineIO_TSC_Charter.md:14]: 👍

**Review by meshula — APPROVED** [review]

**Comment by meshula** [comment]
@ssteinbach I've committed your suggestions. I think to clear the red flags, you need to manually approve the PR.

**Comment by ssteinbach** [comment]
@meshula this is looking good now; I'd still like it if it was rebased before we landed it just to 100% make sure we didn't get some cruft in there.

**Comment by jminor** [comment]
I used the "fetch & merge" button instead of rebase. I think this is ready to merge now. Can you double-check @ssteinbach ?

**Pull request merged**
